### ISSUE TITLE
Any more steps necessary beyond the README instructions?

### ISSUE THREAD

**Issue by dkgaraujo** [apple/tensorflow_macos#5]
Hi all,
First off, congratulations to the developers of this long-awaited interface between tensorflow and the Mac's GPU!
Second, I followed all instructions to the letter (including default location for venv), and still the code is not engaging the GPU. Is there any additional steps that must be taken from the README instructions to start putting the graphics card to use? Please see below the result I obtained. Thanks so much in advance. I'm using an early 2020 16' MBP with AMD Radeon Pro 5500M.

```
import tensorflow as tf
tf.config.list_physical_devices('GPU')
Out[1]: []

tf.test.gpu_device_name()
Out[2]: ''
```

**Comment by nehbit** [comment]
Did you manage to get this working? I'm having the same issue with my Macbook Pro. The graphics card is AMD Radeon R9 M370X.

**Comment by dkgaraujo** [comment]
> Did you manage to get this working? I'm having the same issue with my Macbook Pro. The graphics card is AMD Radeon R9 M370X.

Hi nehbit, yes I did. It turns out that MLCompute will sort of automatically choose the best device for your specific setting. And if you read way down the README file, they mention that you can hard-code which device (CPU or GPU) will be used. So I hard-coded the GPU and it worked. The way you can do this follow below in both python and R:

```
# python
import tensorflow as tf
from tensorflow.python.compiler.mlcompute import mlcompute
mlcompute.set_mlc_device(device_name = 'gpu')

# your amazing neural network model follows...
```

```
# R
reticulate::use_virtualenv(virtualenv = "~/tensorflow_macos_venv/", required = TRUE)
reticulate::py_config()
library(keras)
mlcompute <- reticulate::import("tensorflow.python.compiler.mlcompute")
mlcompute$set_mlc_device(device_name = "gpu")

# your amazing neural network model follows...
```

**Comment by nehbit** [comment]
Thank you! I had done that already, and the GPU still did not show up as an available backend. However once I started running something, I managed to confirm the GPU is working because I can see the GPU being used from Activity Monitor.

It's probably prudent to note for future viewers that this is the case, and that TF behaves exactly like it would be using CPU only, even while GPU is in fact being used. I suspect once this is merged into the main branch the behaviour will improve. 

I'm getting basically zero performance benefit by using GPU over CPU, but there's already an issue to track that: #7.
